script.js

```diff
@@ -411,4 +411,8 @@ function timer() {
         GameState = "End"
     }
 
+}
+
+if (Ball.x <= 0 || Ball.X >= 720) {
+    minus == 0

```

Comment:
change minus to Time

Reply:
I'll add multiple gamemodes down the line